subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -2500,7 +2500,8 @@ static void ticker_update_op_cb(uint32_t status, void *param)
 	/* Reset update requested */
 	ticker_update_ack = ticker_update_req;
 
-#if defined(CONFIG_BT_PERIPHERAL) && (defined(CONFIG_BT_ASSERT) || defined(CONFIG_ASSERT))
+#if defined(CONFIG_ASSERT)
+#if defined(CONFIG_BT_PERIPHERAL)
```

Comment:
Controller does not use `CONFIG_ASSERT`  or `CONFIG_BT_ASSERT`, and assertion checks in the Controller shall/can not be disabled. All `LL_ASSERT` are fatal due to the nature of being in the ISR execution context.

Seems the use has erroneously introduced in d3f386d52b7c00bbdbda044e9439dbecc2088480

```suggestion
#if defined(CONFIG_BT_PERIPHERAL)
```